tests/headers/16-byte-alignment.h

```diff
@@ -27,7 +26,10 @@ struct rte_ipv6_tuple {
         };
 };
 
+// TODO(tmfink) uncomment once test passes
+#if 0
```

Comment:
Thanks for leaving them uncommented in the non-`union` cases so we don't lose test coverage. That is essential to landing this.